src/main/java/apoc/meta/tablesforlabels/OrderedLabels.java

```diff
@@ -0,0 +1,43 @@
+package apoc.meta.tablesforlabels;
+
+import apoc.meta.Tables4LabelsProfile;
+import org.neo4j.graphdb.Label;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Abstraction on an ordered label set, used as a key for tables for labels profiles
+ */
+public class OrderedLabels {
+    List<String> labels;
+
+    public OrderedLabels(Iterable<Label> input) {
+        labels = new ArrayList<>(3);
```

Comment:
I guess it's more concise to use java 8 streams for mapping and sorting?

Reply:
For minimal changes between 3.5 and 4.0 I didn't change it, but if you think updating this would help, let me know. We're in a bit of a time crunch, so unless critical I'd rather leave it.